addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -788,18 +788,22 @@ export function getDeepRange(editable, { range, sel, splitText, select, correctT
     // A selection spanning multiple nodes and ending at position 0 of a node,
     // like the one resulting from a triple click, is corrected so that it ends
     // at the last position of the previous node instead.
-    const endLeaf = firstLeaf(end);
-    const beforeEnd = endLeaf.previousSibling;
-    if (
+    let endLeaf = firstLeaf(end);
+    let beforeEnd = endLeaf.previousSibling;
+    while (
         correctTripleClick &&
-        !endOffset &&
+        (nodeSize(end) && !endOffset) &&
         (start !== end || startOffset !== endOffset) &&
-        (!beforeEnd || (beforeEnd.nodeType === Node.TEXT_NODE && !isVisibleStr(beforeEnd)))
+        (!beforeEnd || (beforeEnd.nodeType === Node.TEXT_NODE && !isVisibleStr(beforeEnd)) || !beforeEnd.isContentEditable)
     ) {
         const previous = previousLeaf(endLeaf, editable, true);
-        if (previous && closestElement(previous).isContentEditable) {
+        if (previous && !closestElement(previous).classList.contains('visually-hidden') && isVisibleStr(previous)) {
```

Comment:
what is this "visually-hidden" class ? where do it come from ?

Reply:
@sebgeelen In the carousel, the 'visually-hidden' class is used on arrow elements, like this: `<span class="visually-hidden o_default_snippet_text">Previous</span>`

Reply:
@sebgeelen waiting for your reply.